sdk/workflow-tengo/src/workdir/index.lib.tengo

```diff
@@ -128,6 +118,10 @@ _fill := func() {
 	values := {}
 	dirs := {}
 
+	queue := undefined
+	cpu := undefined
+	ram := undefined
```

Comment:
Shouldn't we set default values here? At least for queue.
Or add assertion into build() function that queue name is not undefined.
CPU and RAM are OK to be empty, queue is mandatory.

Reply:
It seems you already had to update workdir test in neighbour package because of this new restriction. I still think we need at least validation in `build()` with clear error message.